src/main/java/ch/njol/skript/effects/EffLeash.java

```diff
@@ -64,17 +64,17 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	
 	@Override
 	protected void execute(Event e) {
-		Entity entity = holder.getSingle(e);
-		if (entity == null && !unleash)
+		Entity holder = this.holder.getSingle(e);
+		if (holder == null && !unleash)
 			return;
 		for (LivingEntity target : targets.getArray(e)) {
-			target.setLeashHolder(!unleash ? entity : null);
+			target.setLeashHolder(!unleash ? holder : null);
 		}
 	}
 
 	@Override
 	public String toString(@Nullable Event e, boolean debug) {
-		return unleash ? "un" : "" + "leash " + targets.toString(e, debug) + (unleash ? " from " : "" + " to ") + holder.toString(e, debug);
+		return unleash ? "un" : "" + "leash " + targets.toString(e, debug) + (unleash ? "" : " to ") + holder.toString(e, debug);
```

Comment:
Still missing braces

Reply:
What?

Reply:
(unleash ? "un" : "" + "leash ")

Reply:
What about it?

Reply:
Exactly, you need then but you dont have them here, thats what I've meant since the beginning lol

`return unleash ? "un" : "" + "leash " + targets.......` - your code

Reply:
This doesn't exist bro...